### PR TITLE
feat: add protos for ListSigningKeys

### PR DESCRIPTION
**Pull request by tylerburdsall**
First step for `ListSigningKeys` support

### CODE REVIEW

File: proto/controlclient.proto [hunk 1]
```diff
@@ -55,3 +56,17 @@ message _RevokeSigningKeyRequest {
 
 message _RevokeSigningKeyResponse {
 }
+
+message _SigningKey {
+  string key = 1;
```
Comment (virratanasangpunth) [inline proto/controlclient.proto:61]: `key_id` you mean?

Comment (tylerburdsall) [reply proto/controlclient.proto:61]: Yup, thank you for catching this!

File: proto/controlclient.proto [hunk 2]
```diff
@@ -55,3 +56,17 @@ message _RevokeSigningKeyRequest {
 
 message _RevokeSigningKeyResponse {
 }
+
+message _SigningKey {
+  string key_id = 1;
+  uint64 expires_at = 2;
```
Comment (kvcache) [inline proto/controlclient.proto:62]: Nit: I fully realize nothing else in here is commented. Would you consider putting comments on the fields in these new messages?

File: proto/controlclient.proto [hunk 3]
```diff
@@ -55,3 +56,17 @@ message _RevokeSigningKeyRequest {
...
+}
+
+message _ListSigningKeysRequest {
+  string next_token = 1;
```
Comment (kvcache) [inline proto/controlclient.proto:66]: I remain saddened that we don't use streaming responses for these.

**Review by kvcache — COMMENTED** [review]
nits no blockers

File: proto/controlclient.proto (hunk 2 above)
Comment (tylerburdsall) [reply proto/controlclient.proto:62]: Sure thing!

File: proto/controlclient.proto (hunk 3 above)
Comment (tylerburdsall) [reply proto/controlclient.proto:66]: > I remain saddened that we don't use streaming responses for these.

Yeah, I would like that too and it seems like we aren't using pagination atm. I wanted to keep this here to at least maintain similar patterns

**Review by kvcache — APPROVED** [review]

**Pull request merged**